stable/opa/templates/deployment.yaml

```
@@ -24,14 +43,45 @@ spec:
             - "--config-file=/config/config.yaml"
             - "--tls-cert-file=/certs/tls.crt"
             - "--tls-private-key-file=/certs/tls.key"
+            - "--authentication=token"
+            - "--authorization=basic"
+            - "/authz/authz.rego"
+            - "--ignore=.*"
             - "--addr=0.0.0.0:443"
+{{- if .Values.mgmt.enabled }}
+            - "--addr=127.0.0.1:8181"
```

```suggestion
            - "--addr=http://127.0.0.1:8181"
```

If you don't specify the http scheme here, the server is going to listen for HTTPS connections since TLS credentials have been given.

---

does kube-mgmt support unix sockets for the url? If so, we can pass a unix socket via emptydir and skip the network piece entirely.